RecoLocalCalo/HGCalRecAlgos/plugins/alpaka/HGCalRecHitCalibrationAlgorithms.dev.cc

```diff
@@ -0,0 +1,198 @@
+#include <cstddef>
+
+// CMSSW imports
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+// Alpaka imports
+#include "HeterogeneousCore/AlpakaInterface/interface/traits.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+// HGCal imports
+#include "RecoLocalCalo/HGCalRecAlgos/interface/alpaka/HGCalRecHitCalibrationAlgorithms.h"
+#include "DataFormats/HGCalDigi/interface/HGCalRawDataDefinitions.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  using namespace cms::alpakatools;
+
+  //
+  struct HGCalRecHitCalibrationKernel_flagRecHits {
+    template <typename TAcc>
+    ALPAKA_FN_ACC void operator()(TAcc const& acc,
```

Comment:
```suggestion
    ALPAKA_FN_ACC void operator()(Acc1D const& acc,
```

Reply:
(here and below, this is just a suggestion, not a requirement)

Reply:
OK